src/js/actions/index.js

```diff
@@ -531,13 +531,13 @@ export function fetchShipmentStatusCodes() {
 }
 
 
-export function fetchRequisitionStatusCodes() {
+export function fetchRequisitionStatusCodes(isRequestsList = false) {
   return (dispatch) => {
-    apiClient.get('/openboxes/api/stockMovements/requisitionsStatusCodes')
+    apiClient.get(`/openboxes/api/stockMovements/requisitionsStatusCodes?isRequestsList=${isRequestsList}`)
```

Comment:
let's use `params` attribute to pass parameters instead of template literals
```suggestion
    apiClient.get('/openboxes/api/stockMovements/requisitionsStatusCodes', {
        params: { isRequestsList },
    })
```